cmd/webrtc-load-tester/roles/orchestrator.go

```diff
@@ -138,7 +140,7 @@ func runLoadTest(ctx context.Context, args loadTestArguments) error {
 	}
 
 	glog.Infof("Stream created: %s", stream.ID)
-	glog.Infof("Access the stream at: %s", path.Join(args.APIServer, "/dashboard/streams", stream.ID))
+	glog.Infof("Access the stream at: https://%s", path.Join(args.APIServer, "/dashboard/streams", stream.ID))
```

Comment:
This is really helpful... 
Maybe we could print out the links to `Google Cloud Run`, `Google big Table` with viewership metrics, and `Grafana` dashboard if they are easy to construct